components/CreateAccountForm/index.js

```diff
@@ -1,31 +1,68 @@
+// import { useState } from "react";
```

Comment:
Per Joel:
"So with this(client side auth), we should implement the sign-in form as a React controlled component.
**Plus the create account form.
This will give us the values we need to pass off to the firebase api methods.
React apps should be done within confines of React. React components operate within a "virtual DOM" and sync to the real DOM via async life cycle methods. This makes mixing vanilla and React a risk for race conditions and state mishaps."